iosTraining-Reeen/WeatherViewController.swift

```diff
@@ -62,24 +62,44 @@ class ViewController: UIViewController {
         return stackView
     }()
     
+    private var weatherService: WeatherServiceProtocol
+    
+    init(weatherService: WeatherServiceProtocol) {
+        self.weatherService = weatherService
+        super.init(nibName: nil, bundle: nil)
+    }
+    
+    required init?(coder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+    
     override func viewDidLoad() {
         super.viewDidLoad()
         setupViews()
     }
 }
 
-private extension ViewController {
+private extension WeatherViewController {
+    func displayWeatherCondition() {
+        let conditionString = self.weatherService.getWeatherInformation()
+        weatherConditionImageView.image = imageForWeatherCondition(conditionString)
```

Comment:
```suggestion
        weatherConditionImageView.image = weatherImage(for: conditionString)
```

みたいな API になってると読みやすそうです。